scripts/ci/diff.sh

```diff
@@ -93,12 +93,12 @@ build_output_cmd () {
 format_cost () {
   cost=$1
     
-  if [ -z "$cost" ] | [ "${cost}" == "null" ]; then
```

Comment:
I made these changes because https://www.shellcheck.net/ was flagging them - they were also causing warnings in the bash version that Ubuntu runs (with our Azure DevOps integration uses).